Scripts/Download_Tracking.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import gitlab
+import glob
+import os
+import shutil
+import zipfile
+
+from subprocess import call
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-t", "--token",  required=True,  help="Provide a GitLab Authentication Token")
+parser.add_argument("-v", "--verbose", required=False, action='store_true', help="If used, this script will list the builds it's examining along with their status for users to check what was available")
+
+args = parser.parse_args()
+
+trackingZip         = "./Tracking_Build.zip"
+trackingUnzippedDir = "./Tracking_Build"
+
+searchCount = 500
+
+gitlabToken = args.token
+
+gl = gitlab.Gitlab('https://gitlab.ultrahaptics.com', private_token=gitlabToken)
+
+if os.path.isfile(trackingZip):
+	print("Removing previous tracking zip file")
+	os.remove(trackingZip)
+
+if os.path.isdir(trackingUnzippedDir):
+	print("Removing Unzipped tracking Directory")
+	shutil.rmtree(trackingUnzippedDir)
+
+print("Downloading Artifacts from GitLab")
+
+# Tracking Project ID: 455
+# 5.4 Windows Production Release Job ID: 501756
+
+TrackingProject = gl.projects.get(455)
+
+ArtefactJob = TrackingProject.jobs.get(501756)
```

Comment:
Would it be worth having this Job ID parameterised? Just so that when we update to the next version it's a variable in gitlab instead of in this script

Reply:
Threw this change in to shift this jobID to our CI file so it's easier to change in future